How to contribute

```diff
@@ -0,0 +1,30 @@
+This documentation is primarily aimed at Plaid employees who would like to edit the collection. For instructions on using the collection, see the README.
+
+#### Editing the Plaid Postman collection
+The easiest way to edit the collection is via the Postman app. After downloading it, you can open the Plaid Postman collection and make edits. This is a pretty intuitive process, with a few things to be aware of:
+
+Make sure to use the "tests" tab of the collection when appropriate to save values that will be used as input to other API calls. For example, when calling `/link/token/create`, we have the following under "tests":
+
+```
+let response = pm.response.json();
+linkToken = response.link_token;
+pm.environment.set("link_token", linkToken);
+```
+
+This code allows other API calls to reference ``{{link_token}}`` in the request body rather than requiring the user to copy and paste the Link token.
+
+Make sure to rename examples to "OK" rather than using the default name.
+
+#### Publishing the Plaid Postman collection
+
+**Important: Before you publish the collection, make sure to clear the values of all environment variables except for the ones that were set before you downloaded the collection.**
```

Comment:
I actually don't think you need to do this if they're being stored in the "CURRENT_VALUE" field. But better safe than sorry I guess.

Reply:
I think I've been told once that I did -- maybe Postman has improved the handling since then?